internal/app/backend/backend_service.go

```diff
@@ -168,9 +165,14 @@ func synchronizeRecv(ctx context.Context, syncStream synchronizerStream, m *sync
 			cancelMmfs()
 		}
 
-		if match, ok := m.Load(resp.GetMatchId()); ok {
-			telemetry.RecordUnitMeasurement(ctx, mMatchesFetched)
-			err = stream.Send(&pb.FetchMatchesResponse{Match: match.(*pb.Match)})
+		if v, ok := m.Load(resp.GetMatchId()); ok {
+			match, ok := v.(*pb.Match)
```

Comment:
I don't think this type check is necessary.  It will always be a match because that's the only thing put into the map.  It's reasonable to expect tests to cover this kind of stuff.

Reply:
lint blocked me from any types of direct casting
```
internal/app/backend/backend_service.go:169:11: Error return value is not checked (errcheck)
                        match, _ := v.(*pb.Match)
```

Reply:
just do `match := v.(*pb.match)` ?

Reply:
still doesn't work.. lint is smart :(